views/install.sh.erb

```diff
@@ -171,6 +171,13 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
+  platform=$ID
+  if test "x$platform" = "x"; then
+    platform="linux"
+  fi
```

Comment:
According to http://www.freedesktop.org/software/systemd/man/os-release.html the default value for `ID` is `'linux'` if not otherwise specified. I agree that default is kind of useless for our purposes. The platform I'm interested in correctly sets this value - should I abort instead of having this fallback case?